{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,65 @@
+# {{ cookiecutter.friendly_name }}
+
+[![pypi](https://img.shields.io/pypi/v/hypermodern-python.svg)](https://pypi.org/project/hypermodern-python/)
```

Comment:
Thanks for the update @cauliyang 

These images need to be templatized, there should be no "hypermodern-python".

Also it seems that the alt text is not the same as in the .rst version.

Thanks!

Reply:
@cauliyang @cjolowicz 

It was not only in README.md but in other files, I have created a PR adding those changes.

Reply:
Hi @cjolowicz, 

Thanks for your review 😄 ! 

That is my fault due to my carelessness.  I have fixed that and tried to use an elegant way to change the alt text.   I tested GitHub, PyPI, readthedocs as well. All works well.

Reply:
Thanks for your help! @eyllanesc 


You help me identify what is wrong. But you change 'title' instead of the alt text.  Anyway, Thanks for your time and help:heart: !